elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -276,8 +276,12 @@ private void addFieldMapping(JsonObject payload, BridgeDefinedField bridgeDefine
 			}
 		}
 		else {
-			if ( SpatialHelper.isSpatialFieldLatitude( simpleFieldName ) ) {
-				// we only add the geo_point once
+			if ( SpatialHelper.isSpatialFieldLongitude( simpleFieldName ) ) {
+				// we ignore the longitude field, we will create the geo_point mapping only once with the latitude field
+				return;
+			}
+			else if ( SpatialHelper.isSpatialFieldLatitude( simpleFieldName ) ) {
+				// we only add the geo_point for the latitude field
 				JsonObject field = new JsonObject();
 
 				field.addProperty( "type", "geo_point" );
```

Comment:
Ok, I see. Should we then (probably in a separate issue, though) offer an option to make that first phase work with actual ES geohashes?